### ISSUE TITLE
make dependencies

### ISSUE THREAD

**Issue by pajakgdynia** [ibooij/big-ring#29]
Move all dependencies related to external libraries (directories, versions) to non tracked files.
Change rule with copying external dll's to better form (make rule). Copying these files each time is not necessary and take a lot of time.

**Comment by ibooij** [comment]
I agree. This problem is only there on Windows (for now). I don't really know what the best solution for this would be. QMake isn't really helping, or perhaps I'm just not really understanding QMake that well.
